src/app/store.ts

```diff
@@ -1,6 +1,13 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import todosReducer from '../features/todos';
+import filterReducer from '../features/filter';
+import currentTodoReducer from '../features/currentTodo';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices({
+  todos: todosReducer,
+  filter: filterReducer,
+  currentTodo: currentTodoReducer,
+});
```

Comment:
The function `combineSlices` is not a valid function in `@reduxjs/toolkit`. You should use `combineReducers` instead to combine your reducers. Replace `combineSlices` with `combineReducers`.